### DISCUSSION TITLE
How do I update auto-cpufreq?

### DISCUSSION THREAD

**Discussion by sakibulalikhan** [AdnanHodzic/auto-cpufreq#379] [Q&A]
Hey, great guys!
If **auto-cpufreq** comes **updated** then how can I make _update_ it on **my system**?

**Comment by AdnanHodzic** [comment]
If you install it using [Snap package](https://github.com/AdnanHodzic/auto-cpufreq/#snap-store), then you don't have to worry about anything, everything will be done for you automatically. 

If you install it using [auto-cpufreq-intsaller](https://github.com/AdnanHodzic/auto-cpufreq/#auto-cpufreq-installer) then I suggest you star the [auto-cpufreq github project](https://github.com/AdnanHodzic/auto-cpufreq/) which will keep you informed about things that are happening with the project. 

If you're only interested in releases, you can do this by clicking on "Notifications" button > "Custom" and then checking "Releases". This way whenever there's a new release you'll be notified. 

Then in directory where you cloned `auto-cpufreq` repo, simply run `git pull` followed by `sudo ./auto-cpufreq-installer --install` to install latest changes and then `sudo auto-cpufreq --install` to apply those latest changes.
